### PR TITLE
Change way to provide IERC20

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

**Comment by rzadp** [comment]
Why?

**Comment by xitronix** [comment]
@rzadp remove dependency from another projects for move EnoghtTokenValidator to commons

File: universal-login-relayer/lib/integration/ethereum/validators/EnoughTokenValidator.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const hasEnoughToken = async ({gasToken, gasPrice, gasLimit}: PaymentOpti
     return walletBalance.gte(utils.bigNumberify(gasLimit).mul(gasPrice));
   } else {
     ensure(await isContract(provider, gasToken), InvalidContract, gasToken);
-    const token = new Contract(gasToken, ERC20.interface, provider);
+    const token = new Contract(gasToken, new utils.Interface(IERC20.abi), provider);
```
Comment (JustynaBroniszewska) [inline universal-login-relayer/lib/integration/ethereum/validators/EnoughTokenValidator.ts]: Why not `IERC20.interface`?

Comment (xitronix) [reply universal-login-relayer/lib/integration/ethereum/validators/EnoughTokenValidator.ts]: Didn't work in that way

**Pull request merged**